modules/local/get_calcuts_params.nf

```diff
@@ -1,10 +1,10 @@
 process GET_CALCUTS_PARAMS {
     label 'process_single'
 
-    conda "conda-forge::python"
+    conda "conda-forge::python=3.9"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/python:3' :
-        'docker.io/python:3.9' }"
```

Comment:
This container definition worked for nextflow run, probably not for nf-core download?

I tried to run the test on `dev` branch yesterday again without problems. 
https://github.com/sanger-tol/genomeassembly/actions/runs/8988294821

Reply:
I can't see any difference with the Treeval CI and container definition.
On the farm, `nf-core download` works for me.

Or maybe a version incompatibility ? (e.g. different version of nf-core or apptainer/singularity or docker)

Reply:
That is what we discussed yesterday in the pipeline meeting. 
Obviously `-l` flag didn't help.  Now we can try to fix the `nf-core` version as the one on local.

Reply:
Oki. I'll let you try that